packages/api/src/external/carequality/command/cq-directory/search-cq-directory.ts

```diff
@@ -2,18 +2,21 @@ import { Patient } from "@metriport/core/domain/patient";
 import { Coordinates } from "@metriport/core/external/aws/location";
 import convert from "convert-units";
 import { Sequelize } from "sequelize";
+import { Config } from "../../../../shared/config";
 import { CQDirectoryEntryModel } from "../../models/cq-directory";
 
 export const DEFAULT_RADIUS_IN_MILES = 50;
+const cqExcludeList: string[] = constructGatewayExcludeList();
 
 export type CQOrgBasicDetails = {
   name: string | undefined;
   id: string;
   lon: number | undefined;
   lat: number | undefined;
-  urlXCPD: string;
+  urlXCPD: string | undefined | null;
```

Comment:
Can we not use `null` here? Could convert from `undefined` to `null` if needed - [reference](https://www.notion.so/metriport/Development-Best-Practices-bb3574d3c83d46dc9914a9e5c5542ca7?pvs=4#51c298a38cba4897a9fa639218e69833).